crypto/getenv.c

```diff
@@ -91,7 +135,18 @@ char *ossl_safe_getenv(const char *name)
 #if defined(__GLIBC__) && defined(__GLIBC_PREREQ)
 # if __GLIBC_PREREQ(2, 17)
 #  define SECURE_GETENV
-    return secure_getenv(name);
+#   /* 
+     * Non-default option to permit selected additional variables to be read where 
+     * only the netcap capability is added, rather than follow the standard
+     * priviliged default of disallowing access. (libc > 2.17, linux only)
+     */
+#   if defined(OPENSSL_NETCAP_ALLOW_ENV) && defined(__linux__)
+     /* Use alternate implementation which allows some exceptions if in a secure environment */
+       return(restricted_getenv(name));
```

Comment:
This line doesn't align with the comment above.

In general, there are lots of whitespace issues, including
- missing space after `,`
- missing space either side of binary operators
- opening braces should be on the same line as `if`, `for`, etc
- unnecessary whitespace before `[]` and between empty `[]`
- unnecessary whitespace after `*` in `char * permitted_env`

Reply:
I've made the changes I've noticed. Will keep this conversation open and review https://www.openssl.org/policies/technical/coding-style.html - thanks for the reminder.